serverless-heaven/serverless-webpack
# Fix tests with Serverless v3

The new serverless class must have 3 options for its argument.

```js
// before
serverless = new Serverless();

// after
serverless = new Serverless({ commands: ['print'], options: {}, serviceDir: null });
```

Also:
- remove useless `prettier-eslint-cli` which isn't compatible with ESLint 8 and also isn't necessary (we can use `prettier` directly)
- move all `*.tests.js` file into the `tests/` folder (easier to find them)
- add node 16 to test build
- add build with Serverless v2
- run `npm run format` (that's there is a lot of small changes)

See https://github.com/serverless-heaven/serverless-webpack/pull/1061
